### PR TITLE
Update using-embeds-in-messages.md

### PR DESCRIPTION
**Pull request by zophiale**
v12 upgrade

### CODE REVIEW

File: first-bot/using-embeds-in-messages.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ const embed = new Discord.RichEmbed()
    * Takes a Date object, defaults to current date.
    */
   .setTimestamp()
-  .setURL("https://discord.js.org/#/docs/main/indev/class/RichEmbed")
+  .setURL("https://discord.js.org/#/docs/main/indev/class/MessageEmbed")
```
Comment (cfanoulis) [inline first-bot/using-embeds-in-messages.md]: this URL doesn't look right

**Review by cfanoulis — COMMENTED** [review]
Please check this \/

File: first-bot/using-embeds-in-messages.md (hunk 1 above)
Comment (zophiale) [reply first-bot/using-embeds-in-messages.md]: fixed

**Review by cfanoulis — APPROVED** [review]
LGTM!

**Pull request merged**